options.cpp

```diff
@@ -89,7 +89,7 @@ OpenCLArgList *OpenCLOptTable::ParseArgs(const char *szOptions,
     }
 
     unsigned prev = index;
-    Arg *pArg = ParseOneArg(*pArgs, index);
+    Arg *pArg = ParseOneArg(*pArgs, index).release();
```

Comment:
will "Arg *pArg" be released later?

Reply:
It is because API change from 
`Arg *ParseOneArg(const ArgList &Args, unsigned &Index,
                   unsigned FlagsToInclude = 0,
                   unsigned FlagsToExclude = 0) const;` 
to
 `std::unique_ptr<Arg> ParseOneArg(const ArgList &Args, unsigned &Index,
                                   unsigned FlagsToInclude = 0,
                                   unsigned FlagsToExclude = 0) const;`
There are compiling errors: cannot convert ‘std::unique_ptr<llvm::opt::Arg>’ to ‘llvm::opt::Arg*’ in initialization.

Reply:
please check whether pArg is released later.

Reply:
Update patch, please review it, thanks very much.